### ISSUE TITLE
Gas Fees for `Reply` and `Like`

### ISSUE THREAD

**Issue by borisisnear** [DevHub-DevRels/DevHub-Forum#2]
Users shouldn't have to pay gas to `Reply` and `Like` to a post

**Comment by PiVortex** [comment]
Can do this with meta transactions to DevHub, have to be cautious of people abusing this, and spamming to charge lots of gas. Resolve by setting the size of posts / character limit

**Comment by borisisnear** [comment]
Yeah, I worry about people abusing it. Nevertheless, I think this is being discussed, but I am not sure of the current status
